earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/common/utils.py

```diff
@@ -328,9 +328,17 @@ def RunCmd(os_cmd):
   try:
     if isinstance(os_cmd, str):
       os_cmd = shlex.split(os_cmd)
-    results = subprocess.check_output(os_cmd)
+    p = subprocess.Popen(os_cmd, shell=False,
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.PIPE)
+    results = p.communicate()[1]
+    return_code = p.returncode
+    if return_code != 0:
+      errorMessage = "%s (return code %d)" % (results, return_code)
+      results = errorMessage
```

Comment:
Minor: The old function returned `["", errorMessage]` on error, this one returns `errorMessage.split()`.  I don't really know what the significance of the initial blank string is, nor does it seem a very readable convention, but I hope code that calls this function won't misinterpret when a command succeeds.

Reply:
Suggestion: The user may benefit from seeing the standard error when a command fails.

Reply:
Not sure you meant about the standard error ?

Reply:
I restored the old code behavior in case of an error. It is now returning ["", errorMessage].

Reply:
Not sure about what you meant by standard error ? when the comment fails because of an error or an exception an error is returned to the client code.